src/data/docs/language-reference/README.md

```diff
@@ -911,10 +911,10 @@ The type `{Store v}` means that the computation which results in that type requi
 A constructor `{A} T` for some ability `A` and some type `T`  (or a function which uses such a constructor), can only be used in a scope where the ability `A` is provided. Abilities are provided by `handle` expressions:
 
 ``` unison
-handle h in x
+handle e with h
 ```
 
-This expression gives `x` access to abilities handled by the function `h` which must have the type `Request A T -> T` if `x` has type `{A} T`. The type constructor `Request` is a special builtin provided by Unison which will pass arguments of type `Request A T` to a handler for the ability `A`.
+This expression gives `e` access to abilities handled by the function `h` which must have the type `Request A T -> T` if `e` has type `{A} T`. The type constructor `Request` is a special builtin provided by Unison which will pass arguments of type `Request A T` to a handler for the ability `A`.
```

Comment:
I think strictly speaking the result type doesn’t also have to be T.  Like it could be `Request A t -> Optional t` or `Request A DiscreteDist -> Float` or the result could also carry an ability constraint etc.

Reply:
It’s okay if you don’t want to fix this though, since it’s an independent problem.

Reply:
Ah, good point. This section of the guide tripped me up when I was first learning about abilities. I'll fix it.